v0.12/debian/entrypoint.sh

```diff
@@ -21,8 +9,20 @@ if [ -r $DEFAULT ]; then
     set +o allexport
 fi
```

Comment:
Is this sourcing code _just_ to set (the now-removed) `FLUENTD_CONF` and `FLUENTD_OPT` variables or does it have other uses?

Reply:
No idea.  I guess this is one for @repeatedly.

Reply:
>  does it have other uses?

Fluentd users sometimes set own environment variable via this mechanizm.
So this image keeps same mechaznim.